main.nf

```diff
@@ -3353,7 +3315,60 @@ def svdb_merge_panel_version(task) {
 	cat <<-END_VERSIONS > ${task.process}_versions.yml
 	${task.process}:
 	    svdb: \$( echo \$(svdb) | head -1 | sed 's/usage: SVDB-\\([0-9]\\.[0-9]\\.[0-9]\\).*/\\1/' )
-	    samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
+	END_VERSIONS
+	"""
+}
+
+process postprocess_merged_panel_sv_vcf {
+	cpus 2
+	tag "$group"
+	publishDir "${OUTDIR}/sv_vcf/merged/", mode: 'copy', overwrite: 'true', pattern: '*.vcf'
+	time '1h'
+	memory '1 GB'
+
+
+	input:
+		set group, id, file(merged_vcf) from ch_postprocess_merged_panel_sv
+		set group, id, file(melt_vcf) from ch_melt_vcf
+
+	output:
+		set group, id, file("${group}.merged.bndless.genotypefix.melt.vcf") into vep_sv_panel, annotsv_panel
+		set group, file("${group}.merged.bndless.genotypefix.melt.vcf") into loqusdb_sv_panel
+		set group, file("*versions.yml") into ch_postprocess_merged_panel_sv_vcf_versions
+
+
+	script:
+		"""
+		# Remove BNDs
+		grep -v "BND" $merged_vcf > ${group}.merged.bndless.vcf
```

Comment:
Something to consider for furture work (and that I get constantly nagged about in by bash-scripts by my bash linter).

What do you think about wrapping the bash variables with quotes? `"$merged_vcf"`? To avoid unexpected behaviour for instance if you (for some reason) have spaces or asteriskts in the file name

Viktor had some weird issue last year related to this I think

https://dev.to/jimmymcbride/bash-variables-to-quote-or-not-to-quote-3c6o

(We should not use this type of file names anyway, but anyway!)

Reply:
I.e. not for this specific PR, but a more general consideration